lib/Backends/Interpreter/InterpreterDeviceManager.cpp

```diff
@@ -51,6 +51,17 @@ bool InterpreterDeviceManager::isMemoryAvailable(uint64_t estimate) const {
   return maxMemoryBytes_ >= (usedMemoryBytes_ + estimate);
 }
 
+DeviceInfo InterpreterDeviceManager::getDeviceInfo() const {
+  // TODO: these may need to be tweaked depending on specific CPU
+  DeviceInfo info = DeviceInfo();
+  info.sramCapacity = 256 * 1024 * 1024;
+  info.peakCompute = 2.2 * 1024 * 1024 * 1024;
```

Comment:
Same here. If the number should be hard coded, could you please add some reference?

Reply:
I think this is a good idea but can we defer it?  We currently have no ability to pass configs via onnxifi anyways, which is where we most need these features.

Reply:
Sure, it is OK to defer it.  Reading backend info from config file is one way, and another way is manually setting it before calling HostManager.  If the value here is kind of "fixed" for most of case, I think this function could be renamed as "getDefaultDeviceInfo()", or just set those values by default when the DeviceManager is created.

Reply:
Added to summary